recipes/bfiocpp/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -eux # Abort on error.
+
+PY_ABIFLAGS=$(python -c "import sys; print('' if sys.version_info.major == 2 else sys.abiflags)")
+PY_ABI=${PY_VER}${PY_ABIFLAGS}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+    if [[ -d "$CONDA_BUILD_SYSROOT/usr/include/curl" ]]; then
+        mv "$CONDA_BUILD_SYSROOT/usr/include/curl" "$CONDA_BUILD_SYSROOT/usr/include/curl.do-not-use"
+    fi
+fi
+
+CMAKE_USE_SYSTEM_LIBS="-DTENSORSTORE_USE_SYSTEM_AOM=ON"
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_AVIF=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_BENCHMARK=ON"
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_BLOSC=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_BROTLI=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_BZIP2=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_CARES=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_CURL=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_DAV1D=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_JPEG=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_LIBLZMA=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_LIBYUV=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_LZ4=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_NGHTTP2=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_NLOHMANN_JSON=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_OPENSSL=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_PNG=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_SNAPPY=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_TIFF=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_WEBP=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_ZLIB=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DTENSORSTORE_USE_SYSTEM_ZSTD=ON" 
+CMAKE_USE_SYSTEM_LIBS+=" -DAVIF_ENABLE_GTEST=OFF" 
+CMAKE_USE_SYSTEM_LIBS+=" -DBENCHMARK_ENABLE_GTEST_TESTS=OFF" 
+CMAKE_USE_SYSTEM_LIBS+=" -DINSTALL_GTEST=OFF" 
+
+CMAKE_ARGS="-DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX $CMAKE_USE_SYSTEM_LIBS" python -m pip install . -vvv
```

Comment:
conda-forge already comes with a pre-set CMAKE_ARGS that contains the necessary values
```suggestion
CMAKE_ARGS="$CMAKE_ARGS $CMAKE_USE_SYSTEM_LIBS" python -m pip install . -vvv
```